live-examples/css-examples/css/word-break.css

```diff
@@ -0,0 +1,5 @@
+#example-element {
+    width:10em;
+    text-align: start;
+    border: solid 1px darkgray;
+}
```

Comment:
Nit: Missing empty last line :-/

Reply:
thanks again, I've finally taken like… 20sec to configure my editor so that it doesn't happen again :)